### DISCUSSION TITLE
Windows OS question.

### DISCUSSION THREAD

**Discussion by levis77** [fairy-stockfish/Fairy-Stockfish#650] [Q&A] [answered]
What is the lowest windows version that Fairly Stockfish is compatible with? I tried in Windows 7 , and it seems that  not working well. Is it only support windows 10 or higher?

**Comment by ianfab** [comment, marked as answer]
The Windows version normally shouldn't be the issue, unless it is a 32 bit version. Other than that a possible explanation could be that the executable you downloaded does not match to your CPU. In that case please try the `x86-64` version.
